src/Cello/mesh_Adapt.cpp

```diff
@@ -396,7 +395,7 @@ void Adapt::print(std::string message, const Block * block, FILE * fp) const
                  ia3[1],it3[1],
                  ia3[2],it3[2]);
       }
-      const int l = 1 << (max_level_ - level);
+      //const int l = 1 << (max_level_ - info.index_.level());
```

Comment:
This is commented out -- should it stay for some reason?

Reply:
I commented this out because it was an unused variable. I'm not sure if there was any reason to keep it... If you think I should delete it, I can do that

Reply:
```suggestion
```
Let's rip off that band-aid...